DEPENDENCIES.md

```diff
@@ -115,6 +115,7 @@ graph LR;
   npm-->npm-profile;
   npm-->npm-registry-fetch;
   npm-->npm-user-validate;
+  npm-->npmcli-agent["@npmcli/agent"];
```

Comment:
This is weird?  Why is this different if the package.json of the cli didn't change?

Reply:
because i forgot to resetdeps. 🤦🏼